Bug fix: GetTime js script (#5160)
demisto/content: Scripts/script-GetTime_CHANGELOG.md

## [Unreleased]

> Fixed a bug where providing date input from context returned current date instead of provided date.

Reviewer: ```suggestion
Fixed an issue where providing date input from context returned current date instead of provided date.
```

> Fixed an issue where providing date input from context returned current date instead of provided date.

Reviewer 2: ```suggestion
Fixed an issue where providing a date input from context returned current date instead of provided date.
```
